ports/co/vcpkg.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "co",
+  "version-string": "2.0.1",
```

```suggestion
  "version-semver": "2.0.1",
```
This might be served by [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver).